packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -257,7 +257,7 @@ export async function sendVerificationEmail({
       expires: new Date(new Date().setHours(168)), // +1 week
       team: {
         connect: {
-          id: connectionInfo.orgId || input.teamId,
+          id: input.teamId,
```

Comment:
Invitation now needs to happen to actual passed down teamId as it could be the org itself or a subteam